modules/priceFloors.js

```diff
@@ -103,7 +103,7 @@ export function getFirstMatchingFloor(floorData, bidObject, responseObject = {})
   // if we already have gotten the matching rule from this matching input then use it! No need to look again
   let previousMatch = utils.deepAccess(floorData, `matchingInputs.${matchingInput}`);
   if (previousMatch) {
-    return previousMatch;
+    return {...previousMatch};
```

Comment:
Will modifications only be made at the top level of this object? This change will only make a shallow copy

Reply:
Yes, this is a single object of only primitive types.

`Float`, `String`, `String`

Reply:
Can use `utils.deepClone()` if you want though! maybe that makes more sense?

Reply:
No, what you have is fine, then